force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +87,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = null;
+        if (a == null) {
+            return result;
+        } else if ( a == 0 ) {
+            result = 'Zero';
+           } else if (a > 0) {
+            result = 'Positive';
+           } else {
+            result = 'Negative';
+           }
+        return result; 
```

Comment:
Good work. For these sorts of problems at work, I prefer to use the replace conditional with guard clause pattern I shared in Slack. 

My solution 
```
 public static String checkNumber(Integer a) {
    
        if (a == null) {
            return null;
        }
        if (a > 0) {
            return 'Positive';
        }
        if (a < 0) {
            return 'Negative';
        }
            return 'Zero';
    }
 ```